src/components/ui/ImageGallery/ImageGallery.tsx

```diff
@@ -0,0 +1,58 @@
+import { Button } from '@faststore/ui'
+import React, { useState } from 'react'
+import { Image } from 'src/components/ui/Image'
+
+import { ImageGallerySelector, ImageZoom } from '.'
+
+export interface ImageElementData {
+  url: string
+  alternateName: string
+}
+
+interface ImageGalleryProps {
+  images: ImageElementData[]
+}
+
+function ImageGallery({ images }: ImageGalleryProps) {
+  const [selectedImageIdx, setSelectedImageIdx] = useState(0)
+  const currentImage = images[selectedImageIdx]
+
+  return (
+    <div>
+      <ImageZoom>
+        <Image
+          src={currentImage.url}
+          alt={currentImage.alternateName}
+          variant="product.details"
+          loading="eager"
+        />
+      </ImageZoom>
+      <ImageGallerySelector itemsPerPage={4}>
+        {images.map((image, idx) => {
+          return (
+            <Button
+              key={idx}
+              data-thumbnail-button
+              className={image.url === currentImage.url ? 'selected' : ''}
+              aria-label={`${image.alternateName} - Image ${idx + 1} of ${
```

Comment:
I'm not sure if we should completely remove it. One is for the image being currently displayed, and other is for the images in the selector. Since we have two different "contexts", I guess it would make sense for the screen reader to describe the image that is currently appearing and the list of images that can be clicked to switch the main image? I do not know much about screen readers as well.

I used this site as a reference:
https://www.rei.com/product/194789/teva-reember-slip-ons-womens

It is listed on [baymard](https://baymard.com/blog/accessibility-benchmark-launch) as having good accessibility (just saying this to explain that it is not a random store):

Also just noticed that I made a mistake - the main (selected) image does not have the second part of the alt (after the `-`). Maybe it would be better if we strip the "- Image x of y" part from the selected image, as it is in the aforementioned store?

Reply:
Good references 😄 

I said that because we have the `alternateName` in three different places:
- In the `Image` inside the `ImageZoom` as `alt`.
- In the `Button` inside `ImageGallerySelector` as `aria-label`.
- In the `Image` inside the `Button` as `alt`.

It could sound a little annoying for screen readers. But no worries, good job worrying about it!

Reply:
Hmm got it. Just now I realized that we have it three times, in which two of these times is the same message. Not sure how can we improve it without sacrificing one `alt` or `aria-label`. Suggestions? @saranicoly, thoughts on this?